rescript/plugin_setup.py

```diff
@@ -464,6 +465,46 @@
 )
 
 
+plugin.methods.register_function(
+    function=replace_taxonomy,
+    inputs={
+        'taxonomy': FeatureData[Taxonomy]
+    },
+    parameters={
+        'taxonomy_replacement_map': Metadata
```

Comment:
make this a metadata column instead, then it will necessarily be a series (can skip the column count validation)

Reply:
Whoa cool! Done!